modules/terraform-aci-access-leaf-interface-policy-group/main.tf

```diff
@@ -36,6 +36,15 @@ resource "aci_rest_managed" "infraRsLldpIfPol" {
   }
 }
 
+resource "aci_rest_managed" "infraRsMacsecIfPol" {
+  count      = var.type != "breakout" ? 1 : 0
```

Comment:
i think this should check macsec policy, not breakout

Reply:
Documentation isn't clear, is macsec supported breakout dynamic breakout ports?